ButterflyMX Demo.xcodeproj/project.pbxproj

```diff
@@ -33,7 +33,7 @@
 		1A8DF4A2225CBAF000AB0393 /* Colors.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1A8DF4A1225CBAF000AB0393 /* Colors.swift */; };
 		1A8DF4A3225CBAF500AB0393 /* Colors.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1A8DF4A1225CBAF000AB0393 /* Colors.swift */; };
 		1AA6FBF9219B2A78002F7168 /* Foundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 1AA6FBF8219B2A78002F7168 /* Foundation.framework */; };
-		82ED20FA19FDBE5D880F6D62 /* Pods_ButterflyMX_Demo.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 357B9DFE1973A43366F77AAC /* Pods_ButterflyMX_Demo.framework */; };
+		5B6D8B8B185E1C27F8D0D8A0 /* Pods_ButterflyMX_Demo_Internal.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 2ECABA56EFEBC255A768F2E4 /* Pods_ButterflyMX_Demo_Internal.framework */; };
```

Comment:
I think there shouldn't be any mentions of internal target, please check

Reply:
fixed